python/py-mdtraj/Portfile

```diff
@@ -29,8 +29,10 @@ if {${name} ne ${subport}} {
     depends_build-append    port:py${python.version}-cython \
                             port:py${python.version}-setuptools
```

Comment:
setuptools should be listed as a `depends_lib` dependency. The package installs `console_scripts` which use `from pkg_resources import load_entry_point` and, therefore, requires it also at runtime